README.rst

```diff
@@ -65,28 +66,35 @@ Settings list
 -------------
 All settings are contained in a ``PUSH_NOTIFICATIONS_SETTINGS`` dict.
 
-In order to use GCM, you are required to include ``GCM_API_KEY``.
+In order to use FCM, you are required to include ``FCM_API_KEY``.
```

Comment:
See comment above.  While GCM is deprecated, it does still work, best to leave those instructions alone and add to them.